Makefile

```diff
@@ -1,63 +1,139 @@
+#
+# Makefile for the Links Programming Language.
+#
+
+# This file contains instructions for building a bleeding-edge version
```

Comment:
Maybe make lines 80 columns wide? In Emacs `C-x f`to set the fill width to 80 (in case it is not set) and then `M-q` on a paragraph. This will require inserting newlines between text and code snippets. Otherwise snippets will be treated as part of a paragraph and wrapped as well.